addresses/.supported-networks.json

```diff
@@ -34,5 +34,9 @@
   "zkevm": {
     "chainId": 1101,
     "block-explorer": "https://zkevm.polygonscan.com"
+  },
+  "base": {
+    "chainId": 8453,
+    "block-explorer": "https://wwstage.basescan.org"
```

Comment:
isn't it https://basescan.org/?

Reply:
I guess it is now that it seems to be live :) Did this when it wasn't.